tests/later-test.ts

```diff
@@ -428,3 +428,22 @@ QUnit.test('can be ran "early" with fake timers GH#351', function(assert) {
 
   fakeClock.tick(5001);
 });
+
+QUnit.test('boundRunExpiredTimers is called once when first timer canceled', function(assert) {
+  let done = assert.async(1);
+
+  let bb = new Backburner(['one']);
+
+  let timer = bb.later(function() {}, 500);
+  bb.cancel(timer);
+
+  let boundRunExpiredTimers = bb._boundRunExpiredTimers;
+
+  bb._boundRunExpiredTimers = function() {
```

Comment:
Why do we need to mock this? Is there a way to make this test fail without mocking?

Said differently: how would the bug have presented to users?

Reply:
error does not manifest to user, it only runs `_boundRunExpiredTimers` twice instead of once, 
can remove test if you want, if you think it is too granular